### ISSUE TITLE
clay-make-defun-at-point ignores metatags

### ISSUE THREAD

**Issue by genmeblog** [scicloj/clay.el#4]
Below form is rendered as a nested vector instead of a table.

```clojure
^{:kindly/kind :kind/table :kindly/hide-code true}
[[:a :b :c]]
```

**Comment by daslu** [comment]
Thanks, @genmeblog, good to know.

It looks like `(thing-at-point 'defun)` does not include the form's metadata. I'm wondering whether we could easily change that behavior. 

By the way, `clay-make-last-sexp` does recognize metatags.

**Comment by genmeblog** [comment]
But this requires to go to the end of the expression which might be less convenient when rendering bigger form from the inside.

**Comment by genmeblog** [comment]
What about `cider-defun-at-point` ?

**Comment by daslu** [comment]
Fixed in version 1.3, thanks @genmeblog!
